### ISSUE TITLE
Improve CoNLL-U2Brat converter

### ISSUE THREAD

**Issue by epageperron** [cdli-gh/mtaac_work#40]
## Summary
It appears that the validator contained in our converter is too strict for our data to run through. 

Simplify https://github.com/cdli-gh/conllu.py so it can easily convert data such as our Form ID field, and accept files with underscores in incomplete fields just like the CoNLL-U format standard is supposed to work.

Task #37 is partly dependent on this.

#### 📈 Progress (0-1): 

See Gantt: http://cdli-dev.org/gantt/mtaac_work/

**Comment by epageperron** [comment]
@khoidt: that would be part of the syntax annotation workflow

**Comment by epageperron** [comment]
@sagar-sehgal  this is fixed now with your work yes?

**Comment by thesagarsehgal** [comment]
Yes, I have made a script to detect the tab separation error in the syntax of the data and then manually solved the issue manually ... also have submitted a Pull Request here ...  https://github.com/cdli-gh/mtaac_gold_corpus/pull/5/ ... Also, have written a script to take only first 4 columns.

....**please don't merge the PR** ... I just saw a big error in my last commit ... have added an extra folder in the commit by mistake
